src/proxysql.rs

```diff
@@ -0,0 +1,240 @@
+use chrono::{DateTime, Local};
+use mysql::{prelude::Queryable, Conn, OptsBuilder};
+
+use crate::{
+    config,
+    hosts::{Host, HostStatus},
+    messages,
+    queries::Query,
+};
+
+const MIRROR_QUERY_TOKEN: &str = "Mirror by readyset scheduler at";
+const DESTINATION_QUERY_TOKEN: &str = "Added by readyset scheduler at";
+pub struct ProxySQL {
+    readyset_hostgroup: u16,
+    warmup_time: u16,
```

Comment:
nit: Units here in the name would be nice, e.g., `warmup_time_s`.

Reply:
Done